fw/http_msg.c

```diff
@@ -1677,7 +1677,7 @@ tfw_h2_msg_cutoff_headers(TfwHttpResp *resp, TfwHttpRespCleanup* cleanup)
 	TfwMsgIter *it = &resp->mit.iter;
 	char* body = TFW_STR_CHUNK(&resp->body, 0)->data;
 	TfwStr *crlf = TFW_STR_LAST(&resp->crlf);
-	char *off = body ? body : crlf->data + crlf->len;
+	char *off = body ? body : crlf->data + (crlf->len - 1);
```

Comment:
What place exactly `off` should point at?

Reply:
Last byte of CRLF or first byte of the body.